### ISSUE TITLE
No such file or directory with certain files

### ISSUE THREAD

**Issue by JWL16038** [JWL16038/Music-Classification#8]
In the data labler #7, when opening audio files with long names on Windows, it will throw an `OSException` as in these examples:

```text
[ERROR] 2023-06-04 08:45:24,441 - No such file or directory: F:\AI Projects\Music-Classification\data\raw\classical_music_files\Beethoven\lp_the-nine-symphonies_ludwig-van-beethoven-josef-krips-the-londo_0\disc3\06.02. Symphony No. 3 In E Flat Major, Op. 55 (_Eroica_)_ 4th Movement - Finale _ Allegro Molto_ Poco Andante.mp3
[ERROR] 2023-06-04 08:45:24,467 - No such file or directory: F:\AI Projects\Music-Classification\data\raw\classical_music_files\Beethoven\lp_the-nine-symphonies_ludwig-van-beethoven-josef-krips-the-londo_0\disc5\10.01. Symphony No. 6 In F Major, Op. 68 (_Pastoral_)_ 3rd Movement - Allegro; 4th Movement - Allegro; 5th Movement - Allegretto.mp3
```

**Comment by JWL16038** [comment]
I suspect this might be a encoding issue with certain file names. Will investigate this.

**Comment by JWL16038** [comment]
Not an encoding issue as all paths are ASCII and are readable. This is very likely a long filename issue.

**Comment by JWL16038** [comment]
Bug fixed in this commit fac2bb9
